### ISSUE TITLE
Types are not exported properly in v0.8

### ISSUE THREAD

**Issue by sgmihai** [sealmove/binarylang#28]
As previously discussed:
With binlang 0.8, after removing the * from parsers and fields, I get a undeclared identifier when using a parser in another module, with import. include works.

> This is the code that generated this type declaration
> Should be updated to check the 'visibility' parser option and add export marker (*) accordingly

```
result.add(
  nnkTypeSection.newTree(
    nnkTypeDef.newTree(
      tdef,
      newEmptyNode(),
      if parserOptions.reference: nnkRefTy.newTree(typeBody)
      else: typeBody)))
```

**Comment by sealmove** [comment]
Thanks for posting this. I had forgotten about the issue.
Fix coming soon.

**Comment by sealmove** [comment]
Fixed with 1dc9ea3.
